backend/src/Backend.hs

```diff
@@ -56,13 +63,24 @@ import           Common.OAuth              (OAuthProvider (..),
                                             buildOAuthConfig', oAuthClientIdPath)
 import           Common.Route
 import           Common.Network
+import           Frontend                  (webFrontend)
 
 data BackendCfg = BackendCfg
   { _backendCfg_oAuth                :: OAuthConfig OAuthProvider
   , _backendCfg_getOAuthClientSecret :: Maybe (OAuthProvider -> OAuthClientSecret)
   , _backendCfg_manager              :: Manager
   }
 
+main :: IO ()
+main = do
+  runCheck <- not . null . filter (== "check-deployment") <$> getArgs
+  if runCheck
+     then checkDeployment
+     else runBackend backend webFrontend
+
+frontend :: Frontend (R FrontendRoute)
+frontend = webFrontend
```

Comment:
The intent was that we'd pick here (where we do not need to build under ghcjs, and thus can also have the `desktopFrontend` in scope) between multiple frontends, rather than having dedicated infra in obelisk for that, but that last step is currently under discussion.